R/stage_nldas_ts.R

```diff
@@ -22,7 +24,7 @@
 #' read_ts(files[1])
 #' }
 #' @export
-stage_nldas_ts <- function(sites, var, times, folder = tempdir(), verbose = FALSE, ...){
+stage_nldas_ts <- function(sites, var, times, folder = tempdir(), version=c('tsv','rds'), verbose = FALSE, ...){
   
   if(length(var) > 1) stop("one var at a time, please")
   verify_var_src(var, "nldas", on_fail=warning)
```

Comment:
might make sense to `version <- match.arg(version)` early in this function, to clarify that this function only accepts one version at a time, and to break in an informative place if you try to pass in the wrong thing. i think if you don't specify `version` in the call to `stage_nldas_ts` it should break on the `match.arg` call in `write_ts` (line 70 of this file), but i haven't pulled down your PR to test directly. Is that true, or does it work fine despite my fears?